phpdotnet/phd/Package/PHP/XHTML.php

```diff
@@ -666,7 +666,7 @@ public function format_type_methodparam_text($type, $tagname) {
     }
 
     public function format_type_text($type, $tagname) {
-        $t = strtr(strtolower($type), ["_" => "-", "\\" => "-"]);
+        $t = strtr(ltrim(strtolower($type), "\\"), ["_" => "-", "\\" => "-"]);
```

Comment:
Maybe we should have a function/method which does the ltrim + strtolower ?

Reply:
A "normalize()" function that trims/lowers/replaces.